### DISCUSSION TITLE
Which codespace u used

### DISCUSSION THREAD

**Discussion by alkura** [community/community#69310] [Codespaces]
### Select Topic Area

Question

### Body

i prefeer visual studio

**Comment by fubus05** [comment]
I prefer XCODE

**Comment by fubus05** [comment]
Or VS code

**Comment by fubus05** [comment]
It depends 
from ur work

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
